compat-data.schema.json

```diff
@@ -88,7 +88,11 @@
       "type": "object",
       "properties": {
         "description": { "type": "string" },
-        "mdn_url": { "type": "string", "format": "uri" },
+        "mdn_url": {
+          "type": "string",
+          "format": "uri",
+          "pattern": "https:\/\/developer\\.mozilla\\.org\/docs\/"
```

Comment:
wrt https://spacetelescope.github.io/understanding-json-schema/reference/string.html#pattern:

> When defining the regular expressions, it’s important to note that the string is considered valid if the expression matches anywhere within the string. For example, the regular expression "p" will match any string with a p in it, such as "apple" not just a string that is simply "p". **Therefore, it is usually less confusing, as a matter of course, to surround the regular expression in ^...$, for example, "^p$", unless there is a good reason not to do so.**

... would it be better to have something like:

```
^https:\/\/developer\.mozilla\.org\/docs\/.*$
```

Disclaimer: I'm not very good at regexp.

Reply:
Yes, this pattern would consider "blahblahhttps://developer.mozilla.org/docs/" as valid.

Or: ^https:\/\/developer\.mozilla\.org\/docs\/ 
as .*$ is equivalent to putting nothing at the end of the regexp.

Reply:
Good catch, makes sense to me.
Thanks Will and Jean-Yves!

Reply:
> Or: ^https://developer.mozilla.org/docs/
as .*$ is equivalent to putting nothing at the end of the regexp.

The difference (and the point of the spacetelescope quote) is that this explicitly matches the whole string, rather than just the first part, and this is less confusing, given the arguably strange behavior of `pattern`.